src/widgets/rss/RssWidgetTile.tsx

```diff
@@ -1,44 +1,42 @@
+import Link from 'next/link';
 import {
   ActionIcon,
   Badge,
   Card,
   Center,
-  createStyles,
   Flex,
   Group,
   Image,
   Loader,
-  LoadingOverlay,
   MediaQuery,
   ScrollArea,
   Stack,
   Text,
   Title,
+  createStyles,
 } from '@mantine/core';
-import {
-  IconBulldozer,
-  IconCalendarTime,
-  IconClock,
-  IconCopyright,
-  IconRefresh,
-  IconRss,
-  IconSpeakerphone,
-} from '@tabler/icons';
+import { IconClock, IconRefresh, IconRss } from '@tabler/icons';
 import { useQuery } from '@tanstack/react-query';
 import dayjs from 'dayjs';
 import { useTranslation } from 'next-i18next';
-import Link from 'next/link';
-import { useState } from 'react';
-import { IWidget } from '../widgets';
+
 import { defineWidget } from '../helper';
+import { IWidget } from '../widgets';
 
 const definition = defineWidget({
   id: 'rss',
   icon: IconRss,
   options: {
     rssFeedUrl: {
-      type: 'text',
-      defaultValue: '',
+      type: 'multiple-text',
+      defaultValue: ['https://github.com/ajnart/homarr/tags.atom'],
```

Comment:
Do we really want this default value?

Reply:
The problem is that the way we made the widget it will load even when still in the edit mode from adding it so it won't find a default source and will crash. I think I might fix that as well